flake.nix

```diff
@@ -0,0 +1,62 @@
+{
+  description = "WIP stuff to get started on the pinebook pro.";
+
+  inputs = {
+    nixpkgs.url = "github:NixOS/nixpkgs-channels/nixos-unstable";
+    flake-utils.url = "github:numtide/flake-utils";
+  };
+
+  outputs = { self, nixpkgs, flake-utils }:
+    {
+      overlay = final: prev: import "${self}/overlay.nix" final prev;
+    } // flake-utils.lib.eachDefaultSystem (
+      system:
+        let
+          inherit (nixpkgs) lib;
+          darwin_network_cmds_openssl_overlay = final: prev: {
+            darwin = prev.darwin // {
+              network_cmds = prev.darwin.network_cmds.override { openssl_1_0_2 = prev.openssl; };
+            };
+          };
```

Comment:
???

Is this coming from another of your project's flake and wasn't removed?

I don't expect that building on Darwin would work.

Reply:
I didn't build it, but without this even instantiating the derivation (not sure that's the right terminology) throws an error related to security. I haven't built it (I don't have a Darwin machine on hand) but running `nix flake check` instantiates everything and checks their types. Is there a reason we don't expect it to build on Darwin? If we can confirm it doesn't work then there's no point and I'll remove this and all the Darwin packages, by default I included all platforms nixpkgs supports.